src/scripts/temporaryBattle/TemporaryBattleList.ts

```diff
@@ -282,6 +282,126 @@ TemporaryBattleList['Blue 6'] = new TemporaryBattle(
         imageName: 'Blue2',
     }
 );
+// Kanto Christmas Temporary Battles, based on Blue 2
+TemporaryBattleList['Santa Jynx 1'] = new TemporaryBattle(
+    'Santa Jynx 1',
+    [
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+    ],
+    'Huh ? They are gone by the routes now.',
```

Comment:
This doesn't really make sense, and maybe provide some additional direction:

`Huh? They ran off and are trying to blend it with the local Pokémon population. Find those Santa Jynx and collect more Christmas presents!`

Reply:
Yeah, it looks way better. I put that sentence in. Thanks.